gensrc/script/doris_builtins_functions.py

```diff
@@ -650,6 +649,8 @@
     [['array_exists'], 'ARRAY_BOOLEAN', ['ARRAY_VARCHAR'], ''],
     [['array_exists'], 'ARRAY_BOOLEAN', ['ARRAY_STRING'], ''],
 
+    [['array_first_index'], 'BIGINT', ['ARRAY_BOOLEAN'], 'ALWAYS_NULLABLE'],
+
```

Comment:
this function seems better to return 0 if it is not found in any case, so maybe set always_not_nullable mode

Reply:
done